gateways/authorizenet/authorizenet.go

```diff
@@ -65,7 +65,8 @@ func (client *AuthorizeNetClient) Capture(request *sleet.CaptureRequest) (*sleet
 		return nil, err
 	}
 
-	if authorizeNetResponse.TransactionResponse.ResponseCode != ResponseCodeApproved {
+	if authorizeNetResponse.TransactionResponse.ResponseCode != ResponseCodeApproved ||
+		isAlreadyCaptured(authorizeNetResponse.TransactionResponse) {
```

Comment:
If condition becomes any more complex, will split it out into its own set of approved message response codes